src/services/api.js

```diff
@@ -1,7 +1,15 @@
 const fetchBreakingBad = async () => {
-  const resp = await fetch(`https://www.breakingbadapi.com/api/characters`);
+  const resp = await fetch(`https://www.breakingbadapi.com/api/characters?`);
   const characterData = await resp.json();
-  // this is to fix github CI
   return characterData;
 };
 export default fetchBreakingBad;
+
+// export async function fetchBreakingBad(query) {
```

Comment:
Always clean up old commented out code before commit. Hot tip, if you really want to save it for reference, throw it in Notepad++